### DISCUSSION TITLE
Welcome to cloud-morph Discussions!

### DISCUSSION THREAD

**Discussion by giongto35** [giongto35/cloud-morph#46] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shreyveck** [comment]
hey do anyone know how can deploy this on an outsourced cloud instance and run multiple high-end games connecting it to frontend, How can you develop this a full-stack cloud gaming web app with high-end games like what would be required
